packages/compiler/src/server/vscode-settings.ts

```diff
@@ -0,0 +1,88 @@
+import { Connection, DidChangeConfigurationNotification } from "vscode-languageserver/node.js";
```

Comment:
Feels we can call it client-configuration.ts. No need to couple with vscode, and use configuration to align with the event name

Reply:
and follow the pattern like compiler-serivce